include/nbl/builtin/hlsl/cpp_compat/intrinsics.hlsl

```diff
@@ -206,6 +206,17 @@ inline FloatingPoint rsqrt(FloatingPoint x)
 #endif
 }
 
+template<typename Integer>
+inline Integer bitReverse(Integer base)
+{
+#ifdef __HLSL_VERSION
+	return spirv::bitReverse(base);
+#else
+	return glm::bitfieldReverse(base);
+#endif
+}
```

Comment:
can stay like this until @Przemog1 institutes the amazing convention and rewrite.

Reply:
actually don't call `spirv` directly, call `glsl`

Reply:
that way you can also make sure you're passing the template parameter explictly.

Reply:
@Przemog1 will sort it out